From 792088b73cefdea49dd0e1aee62579f05daeaa8b Mon Sep 17 00:00:00 2001
From: Daniel Wagner <dwagner@suse.de>
Date: Wed, 24 Aug 2022 15:47:55 +0200
Subject: [PATCH kernelshark] kernel-shark: Build missed_event as GUI plugin

missed_events depends on GUI elements hence it should be build as GUI
plugin.

Signged-off-by: Daniel Wagner <dwagner@suse.de>
---

I had some build errors due to global build flags in our build service:

  Linker flags : -flto=auto -Wl,--as-needed -Wl,--no-undefined -Wl,-z,now

 src/plugins/CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/plugins/CMakeLists.txt b/src/plugins/CMakeLists.txt
index e9d906863424..49aee3ed7560 100644
--- a/src/plugins/CMakeLists.txt
+++ b/src/plugins/CMakeLists.txt
@@ -61,8 +61,8 @@ if (Qt5Widgets_FOUND AND TT_FONT_FILE)
 
 endif (Qt5Widgets_FOUND AND TT_FONT_FILE)
 
-BUILD_PLUGIN(NAME missed_events
-             SOURCE missed_events.c MissedEvents.cpp)
+BUILD_GUI_PLUGIN(NAME missed_events
+		 SOURCE missed_events.c MissedEvents.cpp)
 list(APPEND PLUGIN_LIST "missed_events")
 
 install(TARGETS ${PLUGIN_LIST}
-- 
2.37.1

